### PR TITLE
Router scaffolding

### PR DESCRIPTION
**Pull request by ErikMelton**

### CODE REVIEW

File: src/navigation/router.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { IndexPage } from '../pages/home';
+import RegisterPage, { Contact, Optional, Personal, Ticket, TicketDay, TicketLevel, TicketType } from '../pages/register';
+
+import * as ROUTES from './routes'
+
+export const EFRouter = () => (
+  <Router>
+    <NotFoundPage default />
+		<IndexPage path={ROUTES.HOME} />
+		<RegisterPage path={ROUTES.REGISTER} />
```
Comment (angryzor) [inline src/navigation/router.tsx:12]: Just some weird indentation here.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,13 +1,6 @@
...
-		<Localized id="hello"><h1>Hello world!</h1></Localized>
-	</Layout>
-
-export default IndexPage
+import { EFRouter } from "../navigation/router";
+
+export default EFRouter;
+
+export * as register from "./register";
+export * as four_0_four from "./404";
```
Comment (angryzor) [inline src/pages/index.tsx:6]: I'm a bit confused what these are for, since Gatsby would just load the default export, right?

**Review by angryzor — CHANGES_REQUESTED** [review]
Please just remove the semicolons to keep the style consistent. (+ 2 small remarks in line comments)

Otherwise looks good!

File: src/pages/index.tsx (hunk 2 above)
Comment (ErikMelton) [reply src/pages/index.tsx]: Ahhh, I just forgot to remove those 😅

**Review by angryzor — APPROVED** [review]

**Pull request merged**